cirq-core/cirq/contrib/shuffle_circuits/shuffle_circuits_with_readout_benchmarking.py

```diff
@@ -84,6 +101,79 @@ def _generate_readout_calibration_circuits(
     return readout_calibration_circuits, random_bitstrings
 
 
+def _generate_parameterized_readout_calibration_circuit_with_sweep(
+    qubits: list[ops.Qid], rng: np.random.Generator, num_random_bitstrings: int
+) -> tuple[circuits.Circuit, study.Sweepable, np.ndarray]:
+    """Generates a parameterized readout calibration circuit, sweep parameters,
```

Comment:
either add more details to these docstrings or add more comments to the code to provide enough information for people who will read this code a year from now